staging_manifest.yml

```diff
@@ -2,17 +2,18 @@
 applications:
 - name: federalist-staging
 buildpack: https://github.com/cloudfoundry/nodejs-buildpack#v1.5.4
+stack: cflinuxfs2
 domain: fr.cloud.gov
 disk_quota: 2G
-memory: 512MB
+memory: 1G
```

Comment:
No big deal, I would rely on a separate environment variable for staging configuration loading rather than production. `NODE_ENV` is very overloaded and triggers many side effects if it's not set properly.